opsmngr/accesslist_api_keys.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opsmngr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const accessListAPIKeysPath = "api/public/v1.0/orgs/%s/apiKeys/%s/accessList"
+
+// AccessListAPIKeysServiceOp handles communication with the AccessList API keys related methods of the MongoDB Atlas API.
+type AccessListAPIKeysServiceOp service
+
+var _ atlas.AccessListAPIKeysService = &AccessListAPIKeysServiceOp{}
+
+// List gets all AccessList API keys.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/api-access-list/get-all-api-access-entries/
+func (s *AccessListAPIKeysServiceOp) List(ctx context.Context, orgID, apiKeyID string, listOptions *atlas.ListOptions) (*atlas.AccessListAPIKeys, *Response, error) {
+	if orgID == "" {
+		return nil, nil, atlas.NewArgError("orgID", "must be set")
+	}
+	if apiKeyID == "" {
+		return nil, nil, atlas.NewArgError("apiKeyID", "must be set")
+	}
+
+	path := fmt.Sprintf(accessListAPIKeysPath, orgID, apiKeyID)
+	path, err := setQueryParams(path, listOptions)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.Client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	root := new(atlas.AccessListAPIKeys)
+	resp, err := s.Client.Do(ctx, req, root)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	if l := root.Links; l != nil {
+		resp.Links = l
+	}
+
+	return root, resp, nil
+}
+
+// Get retrieve information on a single API Key access list entry using the unique identifier for the API Key and desired permitted address.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/api-access-list/get-one-api-access-entry/
```

Comment:
wrong link?

Reply:
I don't find the API access list in the ops manager doc :/

Reply:
there is only with whitelist https://docs.opsmanager.mongodb.com/current/reference/api/org-api-key-whitelists/

Reply:
this is going to be released with 5.0 I was going to suggest the cloud manager links but not there either, may be worth reaching out to docs to confirm when these will be available, ok to leave blank for now